lightning-invoice/src/payment.rs

```diff
@@ -1665,16 +1632,12 @@ mod tests {
 		let invoice_payer =
 			InvoicePayer::new(&payer, router, &logger, event_handler, Retry::Attempts(0));
 
-		// Succeed 1st path, leave 2nd path inflight
-		let payment_id = invoice_payer.pay_invoice(&payment_invoice).unwrap();
-		invoice_payer.handle_event(Event::PaymentPathSuccessful {
-			payment_id, payment_hash, path: route.paths[0].clone()
-		});
```

Comment:
Is the comment at the beginning of test still accurate with this removed? Same goes for the test name.

Reply:
good catch -- we're just sending two payments here and making sure the second one takes the inflight HTLCs of the first one into account